content/event/20170201-react-workshop.md

```diff
@@ -0,0 +1,40 @@
++++
+date = "2017-02-01"
+description = ""
+tags = ["workshop", "framework", "delavnice", "react"]
+title = "Delavnica: Let's React"
+linkrsvp = "TODO"
+
++++
+**Trajanje:** od 12:00-16:00
+
+**Predava:** Urban Cvek, Študent FRI, Shopster Indoor Solutions
+
+React je popularna JavaScript knjižnica za grajenje spletnih uporabniških vmesnikov. Leta 2011 ga je ustvaril Facebook za svoje potrebe, ter ga leta 2013 objavil kot odprto kodni projekt. V delovanju ga verjetno vsak dan vidite na Facebooku, Instagramu, Netflixu in mnogih drugih spletnih straneh/apikacijah sedaj pa boste lahko spoznali kako ga uporabiti tudi sami.
+
+Predstavljen je kot "V" v "MVC" modelu. Izkaže pa se da je še veliko več. "M" in "C" pa tako ali tako ne potrebujemo saj imamo za to JavaScript.
+
+<!--more-->
+
+React se ne uporablja zgolj za grajenje spletnih aplikacij temveč tudi "native" aplikacij tako za iOS, Android, Windows in macOS. Konec leta 2016 pa je izšla tudi beta verzija dodatka k Reactu za Virtual Reality.
```

Comment:
...spletnih aplikacij, temveč tudi...

.. temveč tudi iOS, Andorid, Windows in mac OS aplikacij.